docs/cudf/source/10min-cudf-cupy.ipynb

```diff
@@ -17,10 +17,8 @@
    "source": [
```

Comment:
<p>Given these timings, why are we recommending (1) over (3)? It looks like it's about 8 times slower. Also are there any differences between <code>%time</code>  and <code>%timeit</code>  results here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/7158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Is this difference just showing the JIT overhead, or is <code>.values </code> slower than <code>as_gpu_matrix</code>?</p>

Reply:
With `%timeit` the differences are less extreme, but `.values` is still the slowest for both `DataFrame` and `Series`.